OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -114,7 +114,7 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 							break;
 						}
 
-						Game.AddChatLine(Color.White, ServerChatName, "The game has started.");
+						Game.AddTransientMessage("The game has started.", Color.White);
```

Comment:
I think this should remain as a chat line because it makes clear when a game has started. This is useful when reviewing the chat history (e.g. in a replay).

Reply:
I agree!